src/sentry/issues/ignored.py

```diff
@@ -19,6 +29,33 @@ def handle_archived_until_escalating(
     metrics.incr("group.archived_until_escalating", skip_internal=True)
     for group in group_list:
         remove_group_from_inbox(group, action=GroupInboxRemoveAction.IGNORED, user=acting_user)
-    # TODO(snigdha): create a forecast for this group
+
+    forecasts = get_forecasts(list(group_list))
+    group_forecasts = []
+    for group, forecast in forecasts:
+        if not forecast:
+            logger.info(
+                "archived_until_escalating.no_forecast_created",
+                extra={
+                    "detail": "No forecast created for groups",
+                    "group_id": group.id,
+                },
+            )
+        else:
+            group_forecasts.append((group, forecast))
+
+    if not group_forecasts:
+        return
+    GroupForecast.objects.filter(group__in=group_list).delete()
+    GroupForecast.objects.bulk_create(
+        [GroupForecast(group=group, forecast=forecast) for group, forecast in group_forecasts]
+    )
```

Comment:
Is the plan to still use this table for now, and transition to use nodestore later on?

Reply:
We could keep using it for the sake of moving things forward (your call Snigdha), however, it should be a priority to move to it ASAP.

Keep close contact with Jodi who's working on it.

Let's not work on writing tests for GroupForecast

Reply:
This change isn't urgent, I'm happy to wait until we've moved to Nodestore.